middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -40,7 +40,6 @@ module VB = Var_in_binding_pos
 (* Do not use [Simple.symbol], use this function instead, to ensure that
    we correctly compute the free names of [Code]. *)
 let use_of_symbol_as_simple acc symbol =
-  let acc = Acc.add_symbol_to_free_names ~symbol acc in
```

Comment:
Why was this removed ? I think it makes sense to track free symbols, and I can't see any other place where it would be added.